### PR TITLE
add a get_num_followers endpoint

### PR DESCRIPTION
**Pull request by chrisndodge**

### CODE REVIEW

**Comment by chrisndodge** [comment]
FYI @antoviaque since we are both working recently on cs_comment_service. I need this information (number of thread followers) for Notifications work.

**Comment by antoviaque** [comment]
@chrisndodge Thanks, it's good to know this is available, this might come in handy in future changes. Do you need a formal review on this? This LGTM aside from needing a test, but if you want an extra pair of eyes on it we can also ask @e-kolpakov to have a look, he wrote the latest PR on adding stats to the comments service.

File: api/comment_threads.rb [hunk 1]
```diff
@@ -105,3 +105,16 @@
     course_stats.to_json
   end
 end
+
+get "#{APIPREFIX}/threads/:thread_id/num_followers" do |thread_id|
+  begin
```
Comment (e-kolpakov) [inline api/comment_threads.rb]: @chrisndodge I'm not quite sure, but most likely this `begin`-`end` pair is not needed here as `do` already creates a block. But my ruby is far from perfect, so I might be wrong.

**Comment by e-kolpakov** [comment]
@chrisndodge +1 for test; other than that LGTM.

**Pull request merged**